sql-select/actor-names.sql

```diff
@@ -0,0 +1,2 @@
+select ("actorId", "firstName", "lastName") from actors;
+returning *;
```

Comment:
`returning` on a SELECT doesn't do anything useful. Also, the `;` at the end of the SELECT statement ends that statement, so the `returning *` here is a new statement, acting on nothing.